ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/component/StateComponent.java

```diff
@@ -41,6 +41,6 @@
      *
      * @param state the component state object
      */
-    void loadState(@NotNull JsonObject state);
+    void setState(@NotNull JsonObject state);
```

Comment:
Just to make names more clear for developer.
The `StateComponent` interface has
`JsonObject getState()` method and now it has `setState` instead of `loadState`